source/partials/_guides-sidebar-nav.html.erb

```diff
@@ -15,16 +15,23 @@
     <div class="navbar-default sidebar" role="navigation">
         <div class="sidebar-nav navbar-collapse">
             <ul class="nav" id="side-menu">
-                <li>
+              <li>
+                    <a href="#">Installation<span class="fa arrow"></span></a>
+                    <ul class="nav nav-second-level">
+                        <li>
+                            <a href="mac-install.html">Mac</a>
```

Comment:
If this branch is off `develop` (or is gonna be rebased), the install guide paths have changed to:

```
├── install-guides
│   ├── dev-tools.html.erb
│   ├── git.html.erb
│   ├── index.html.erb
│   ├── linux.html.erb
│   ├── mac.html.erb
│   └── windows.html.erb
```

Also, guides have moved to:

```
├── guides
│   ├── channels.html.erb
│   ├── index.html.erb
│   ├── persistence.html.erb
│   └── slackir.html.erb
```

Reply:
cool thanks will update